sample_size/sample_size.py

```diff
@@ -0,0 +1,17 @@
+from typing import List
```

Comment:
simply test NumPy and statsmodels are imported correctly, this will be directly overridden soon